bip-MarcoPon-BlockchainURI.mediawiki

```diff
@@ -0,0 +1,80 @@
+<pre>
+  BIP: bip-MarcoPon-BlockchainURI
+  Title: URI scheme for Blockchain references / exploration
+  Author: Marco Pontello <marcopon@gmail.com>
+  Status: Draft
+  Type: Standards Track
+  Created: 29 August 2015
+</pre>
+
+==Abstract==
+
+This BIP propose an URI scheme for looking up blocks, transactions, addresses on a Blockchain explorer, or in general to make proper Blockchain references
+
+==Motivation==
+
+The purpose of this URI scheme is to enable users to handle all the requests for details about blocks, transactions, etc. with their preferred tool (being that a web service or a local application).
+Currently a Bitcoin client usually points to an arbitrary blockchain explorer when the user looks for the details of a transaction (es. Bitcoin Wallet uses BitEasy, Mycelium or Electrum uses Blockchain.info, etc.).
+Other times resorting to cut&paste is needed.
+The same happens with posts and messages that reference some particular txs or blocks, if they provide links at all.
+
+==Specification==
+
+The URI follow this simple form:
+
+ <nowiki>blockchain:[//chain]</type/hash></nowiki>
```

Comment:
You're just looking for a hash that will be unique, so really you need either the genesis or the "fork genesis" hash. If later feathercoin was to fork again, the new coin would be the next fork hash. This way you have a unique identifier for each chain. I suggest keeping exactly the same format, but just explain what `//chain` means, i.e. it is usually the genesis hash, but it could also be a fork hash (citing feathercoin as the example).